lib/pharos/configuration/host.rb

```diff
@@ -77,6 +78,15 @@ def short_hostname
         hostname.split('.').first
       end
 
+      def ssh_options
+        {}.tap do |opts|
+          opts[:keys] = [ssh_key_path] if ssh_key_path
+          opts[:send_env] = [] # override default to not send LC_* envs
+          opts[:proxycommand] = ssh_proxy_command if ssh_proxy_command
```

Comment:
This should do the same as `opts[:proxy] = Net::SSH::ProxyCommand.new(...)`: https://github.com/net-ssh/net-ssh/blob/master/lib/net/ssh/config.rb#L251